bin/ember-template-lint.js

```diff
@@ -172,6 +191,13 @@ function run() {
     }
   }
 
+  if (printPending && filesWithErrors.length > 0) {
+    console.log(
```

Comment:
We should honor `--json` flag as well here, so that its possible to get things in a parsable JSON format:

```js
let pendingList = JSON.stringify(filesWithErrors, null, 2);

if (options.named.json) {
  console.log(pendingList);
} else {
  console.log(
    'Add the following to your `.template-lintrc.js` file to mark these files as pending.\n\n'
  );

  console.log(`pending: ${pendingList}`);
}
```